tests/graphql_newline/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,274 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`directive_decl.graphql 1`] = `
+directive @dir(
+  # comment
+  arg1: String
+
+
+  # comment
+  arg2: String
+  arg3: String
+
+) on Field
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+directive @dir(
+  # comment
+  arg1: String
+
+  # comment
+  arg2: String
+  arg3: String
+) on Field
+
+`;
+
+exports[`directives.graphql 1`] = `
+query MyQuery @directive(
+  arg: 5
+
+  # comment
+  arg2: 10
+
+) {
+  field @skip(
+    if: true
+
+    # comment
+    cursor: 10
+
+  ) @nope
+  otherField
+  ...fragmentSpread, @include(if: ["this isn't even a boolean", "wow, that's really odd",,,,,])
+}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+query MyQuery
+  @directive(
+    arg: 5
+
+    # comment
+    arg2: 10
+  ) {
```

Comment:
This line looks odd to me

Reply:
This was the current behavior when it breaks: [playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBHArnATgTwAQCyOAiprngAIAmAllnGDDQG5wAUAOlHngDbQBzAIJYB6ALYIYAOQCGkpHg4h+UYaIlSAarN6ZlXHqvVjJsOQqUrBI09t36QhvjY1mZ8uIuXHbm2Dp6cAZQAJR4wM4AZjRwvFRcAL4gADQgEAAOTNAAzsigsjkwyDBYmGkARliyYADWcDAAyhk1NGolZXBpVBBgyFG6OV0gbUNYMAAK1QLisv2DwwBWOQAeAELVdQ2NngAybXDzvENpEOgwGecATEcnIC1YY8ggAtUZABaovKn3WG0wAHUaFQYO9kAAOAAMaQyWAgQwBbw65RA9AwdDgU1kMzmSAGx2GQ3ENGRhLaAl4cFIEHgt2GMFkFSBILBSCuaVKshovHJAGEIOJZs8oNBDml0EMACqMvJ4haJRJAA)

Reply:
Yeah. Don't have any better ideas either. Fine to leave it.